release/default.nix

```diff
@@ -127,6 +127,11 @@ let
   # The exclusion list is applied to overlay packages and important packages.
   # Supports excluding packages from nested sets using "dotted names" like "fc.blockdev".
   excludedPkgNames = [
+    "mongodb_3_2"
```

Comment:
can be removed, they are defined in the service directly now and not in the overlay